packages/wrangler/src/bundle.ts

```diff
@@ -76,10 +77,12 @@ export async function bundleWorker(
 		workerDefinitions: WorkerRegistry | undefined;
 		firstPartyWorkerDevFacade: boolean | undefined;
 		targetConsumer: "dev" | "publish";
+		local?: boolean;
```

Comment:
This is an internal function, so let's insist on a value here:

```suggestion
		local: boolean;
```

Then we don't need to default it below.